wiki/Reporting_bad_behaviour/Abuse /vi.md

```diff
@@ -0,0 +1,81 @@
+
+Nếu bạn cảm thấy những nhận xét/đề nghị thực sự không phù hợp, khiếm nhã hoặc bạn là người dùng chưa đủ tuổi vị thành niên, vui lòng gửi email đến [abuse@ppy.sh](mailto:abuse@ppy.sh) với càng nhiều thông tin càng tốt.
+
+## Trong một bài đăng công khai trên diễn đàn của osu!
+
+[Báo cáo bài đăng](/wiki/Reporting_bad_behaviour#forum) cho [Nhóm kiểm duyệt toàn cầu](/wiki/People/Global_Moderation_Team) của chúng tôi càng sớm càng tốt.
+
+Để báo cáo , hãy nhấp vào nút `Report post` ở góc trên bên phải của bài đăng trên diễn đàn.
+
+Ngoài ra, cũng có thể nhấp vào tên người dùng của bất kỳ thành viên GMT nào trong danh sách trên trang wiki trên để đến hồ sơ của họ. Nếu bạn không biết nên chọn ai để báo cáo, [thay vào đó chỉ cần sử dụng liên kết này](https://osu.ppy.sh/users/102335).
+
+Nhấp vào nút có hình phong bì ngay bên dưới lá cờ quốc gia của người dùng. Thao tác này sẽ đưa bạn đến cuộc trò chuyện.
+
+Dán liên kết bài đăng trên diễn đàn chứa nội dung có vấn đề và cho GMT biết một chút về những gì đang diễn ra. Họ sẽ xóa bài đăng cho bạn ngay khi có thể.
+
+Nếu bài đăng vẫn chưa được xóa trong khoảng thời gian hợp lý, vui lòng gửi email đến [abuse@ppy.sh](mailto:abuse@ppy.sh) chứa càng nhiều thông tin càng tốt.
+
+## Trên máy chủ Discord công khai có liên quan đến osu! 
+
+Chặn ngay người dùng bằng cách nhấp chuột phải vào tên người dùng của họ và chọn `Block` xuất hiện trong danh sách thả xuống .
+
+Sau đó, hãy liên hệ ngay với quản trị viên của máy chủ bằng cách tìm ai đó ở đầu danh sách người dùng và cho họ biết điều gì đã xảy ra. Nếu bạn không thể tìm ra ai là quản trị viên, hãy tìm người có vương miện bên cạnh tên của họ - họ là chủ sở hữu máy chủ và chịu trách nhiệm về những gì xảy ra bên trong máy chủ đó.
+
+Nếu hành vi lạm dụng này tiếp tục diễn ra và quản trị viên không làm gì về việc đó, [vui lòng thông báo cho nhóm Discord Trust & Safety bằng cách gửi báo cáo](https://dis.gd/request).
+
+Ngoài ra, nếu bạn biết tài khoản osu! của người làm phiền bạn, vui lòng gửi email đến [abuse@ppy.sh](mailto:abuse@ppy.sh) chứa càng nhiều thông tin thì tốt và mọi thứ sẽ được điều tra kịp thời.
+
+## Trên Twitter được liên kết với tài khoản osu!
+
+Nhấp vào ba dấu chấm ở trên cùng bên phải của tweet và nhấp vào 'Report Tweet'. Hãy đọc lời nhắc kèm theo một cách cẩn thận và làm theo chúng. Bạn cũng có thể đọc [hướng dẫn của Twitter về cách báo cáo hành vi lạm dụng](https://help.x.com/en/safety-and-security/report-abusive-behavior) để biết thêm thông tin về quá trình này.
+
+Sau đó, block hoặc mute người dùng đó bằng cách nhấp vào ba dấu chấm ở trên cùng bên phải dòng tweet của họ (nếu bạn đang sử dụng trang web hoặc ứng dụng dành cho điện thoại di động). Tham khảo [hướng dẫn của Twitter về cách chặn tài khoản](https://help.x.com/en/using-x/blocking-and-unblocking-accounts) để biết thêm thông tin.
```

Comment:
I highly recommend translating "block hoặc mute" to "chặn hoặc tắt tiếng" since Twitter does have Vietnamese but if you want to keep it as is, that's okay